src/tests/ftest/osa/offline_drain.py

```diff
@@ -49,8 +49,11 @@ def run_offline_drain_test(self, num_pool, data=False, oclass=None, pool_fillup=
             oclass = self.ior_cmd.dfs_oclass.value
 
         # For testing multiple ranks as dmg parameters, use a list of ranks.
-        if multiple_ranks is True:
-            self.ranks = self.multiple_ranks
+        if multiple_ranks:
+            ranklist = list(self.server_managers[0].ranks.keys())
+            self.ranks = ",".join(map(str, self.random.sample(ranklist), k=2))
```

Comment:
Not sure what `k=2` is for.

Reply:
`k` should be an arg to the `random.sample` function. This is what pylint is complaining about
```suggestion
            self.ranks = ",".join(map(str, self.random.sample(ranklist, k=2)))
```

Reply:
`k=2` means choose 2 random elements from `ranklist`

Reply:
Oops... Let me check...

Reply:
fixed.